README.md

```diff
@@ -37,6 +37,10 @@ bundle exec jekyll build
 ```sh
 bundle exec jekyll serve
 ```
+```sh
+## To live reload changes
+bundle exec jekyll serve -l
```

Comment:
please prefer long-form option names

and good note!

Reply:
Done!